ignite/metrics/multilabel_confusion_matrix.py

```diff
@@ -0,0 +1,112 @@
+import numbers
+from typing import Callable, Optional, Sequence, Tuple, Union
+
+import torch
+
+from ignite.exceptions import NotComputableError
+from ignite.metrics.metric import Metric, reinit__is_reduced, sync_all_reduce
+from ignite.metrics.metrics_lambda import MetricsLambda
+
+__all__ = [
+    "MultiLabelConfusionMatrix",
+]
+
+
+class MultiLabelConfusionMatrix(Metric):
```

Comment:
Let's add a docstring here. You can try to copy and adapt the one from ConfusionMatrix.

Reply:
Added a docstring.